src/rosdiscover/recover/call.py

```diff
@@ -214,6 +215,9 @@ def eval(self, context: SymbolicContext) -> None:
     def is_unknown(self) -> bool:
         return self.param.is_unknown()
 
+    def to_str(self) -> str:
```

Comment:
use `__str__` instead (you'll need to add `str=False` to your `attr.s` call)

Reply:
fixed